x-pack/plugins/apm/public/components/alerting/ui_components/chart_preview/index.tsx

```diff
@@ -39,13 +39,20 @@ export function ChartPreview({
   uiSettings,
   series,
 }: ChartPreviewProps) {
+  const [yMax, setYMax] = useState(threshold);
+
   const theme = useTheme();
   const thresholdOpacity = 0.3;
   const timestamps = series.flatMap(({ data }) => data.map(({ x }) => x));
   const xMin = Math.min(...timestamps);
   const xMax = Math.max(...timestamps);
   const xFormatter = niceTimeFormatter([xMin, xMax]);
 
+  function updateYMax() {
+    const values = series.flatMap(({ data }) => data.map((d) => d.y ?? 0));
+    setYMax(Math.max(...values, threshold * 1.2));
```

Comment:
What is the meaning of `1.2` here?

Reply:
Thanks @yngrdyn, there was a comment about it in a previous [PR](https://github.com/elastic/kibana/pull/143238/files#diff-80e1f52e10c9186facd40c36d880cab859c872c5f329ad5dfadfe7ff2d0ef532), I will add it back